src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -150,12 +150,18 @@ public function quoteColumnName($columnName)
      */
     public function hasTable($tableName)
     {
-        $tables = array();
-        $rows = $this->fetchAll(sprintf('SELECT table_name FROM information_schema.tables WHERE table_schema = \'%s\';', $this->getSchemaName()));
-        foreach ($rows as $row) {
-            $tables[] = strtolower($row[0]);
-        }
-        return in_array(strtolower($tableName), $tables);
+        $result = $this->getConnection()->query(
+            sprintf(
+                'SELECT *
+                FROM information_schema.tables
+                WHERE table_schema = %s
+                AND lower(table_name) = lower(%s)',
+                $this->getConnection()->quote($this->getSchemaName()),
+                $this->getConnection()->quote($tableName)
+            )
+        );
+
+        return $result->rowCount() !== 0;
```

Comment:
Should/would this be better as ```return $result->rowCount() === 1;``` as we are expecting exactly 1 row for a truthy value. Because this is PHP, ```false`, ```-1```, ```99```, ```'who cares'```, ```null```, would all be true.